CourseGrab/AppDelegate.swift

```diff
@@ -61,13 +61,16 @@ extension AppDelegate: GIDSignInDelegate {
         let credential = GoogleAuthProvider.credential(withIDToken: authentication.idToken, accessToken: authentication.accessToken)
         Auth.auth().signIn(with: credential) { (result, error) in
             User.current?.googleToken = authentication.idToken
+            // Enable notifications when user successfully signs in
+            NetworkManager.shared.enableNotifications(enabled: true).observe { _ in
+            }
```

Comment:
I would put the closing bracket online 65 since it's an empty closure. Also, make sure to set

```
UserDefaults.standard.areNotificationsEnabled = true
```